sktime/forecasting/naive.py

```diff
@@ -37,6 +37,9 @@ class NaiveForecaster(OptionalForecastingHorizonMixin,
                      When sp is not 1, mean of all values
                      in a season from last window will be
                      forecasted for each season.
+        * "drift": forecast the values increasing or
```

Comment:
I'd say: "forecast by fitting a line between the first and last point of the window and extrapolating it into the future"